### ISSUE TITLE
About

### ISSUE THREAD

**Issue by lsssssssssss** [idanachi/DefRec_and_PCM#16]
I couldn't get the visualizations I wanted after the reconstruction.The network doesn't seem to be converging.Could you please send me the weight parameters after pre-training

**Comment by IdanAchituve** [comment]
Hi,
I didn't experience any convergence issues. Which variant of the model are you running? What diverges exactly? 
I don't have the parameters of the learned model.

**Comment by lsssssssssss** [comment]
When I visualized the reconstructed point cloud, I got a jumble of points

**Comment by lsssssssssss** [comment]
How can I get an effect like figure 9-11

**Comment by IdanAchituve** [comment]
As seen in the paper, the reconstruction is not very good as this isn't the main point of the method. However, if the training goes well and the reconstruction loss decreases during training the plots should come out reasonable. I am not sure why you got the results that you did.
